packages/app-mobile/android/app/build.gradle

```diff
@@ -136,6 +136,10 @@ dependencies {
     } else {
         implementation jscFlavor
     }
+
+    // Needed for Whisper speech-to-text
+    implementation 'com.microsoft.onnxruntime:onnxruntime-android:latest.release'
+    implementation 'com.microsoft.onnxruntime:onnxruntime-extensions-android:latest.release'
```

Comment:
- The implementation uses the Kotlin/Java bindings to ONNX. Originally, I tried recording audio in the correct format with Kotlin, sending the recorded data to React Native, then starting Whisper using the ONNX React Native bindings ([implementation](https://github.com/laurent22/joplin/pull/11158/commits/590ff0717b83426c21b7160f4d94e2171ca011af)). However, this was failing with an error similar to the following:
  ```
  max_length > sequence_length was false. max_length (448) shall be greater than input sequence length (80000)
  ```
  Here, the input sequence was the raw audio data being provided to ONNX. `max_length` is the maximum output sequence length.
  - Using the React Native ONNX bindings also requires transferring the audio data from the audio recorder to React Native, then from React Native to ONNX, which adds additional overhead.